.github/workflows/README.md

```diff
@@ -1,60 +1,17 @@
-# Cross version tests
+# GitHub Actions workflows
```

Comment:
> but not sure how will js files interact with yaml files.

`actions/github-script` runs them. The aim of these js files is just to improve the readability and editability. We can directly write JS code in the `script` field but it's much easier to edit JS code as a JS file than edit it on a yaml file because you can use syntax highlighting or code suggestions of your editor.

```yml
name: Notify DCO Failure

on:
  pull_request_target:
    types:
      - opened

jobs:
  notify:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/github-script@v4
        with:
          script: |
            const script = require(
              👇👇👇
              `${process.env.GITHUB_WORKSPACE}/.github/workflows/notify-dco-failure.js`
            );
            script({ context, github });
```

Reply:
Including them in the table like below might clarify what they are:

```md
## Automation

| File                        | Role                                                        |
| :-------------------------- | :---------------------------------------------------------- |
| `labeling.yml`              | Automatically apply labels on issues and PRs                |
| `notify-dco-failure.yml`    | Notify a DCO check failure                                  |
| `notify-dco-failure.js`     | The main script of the `notify-dco-failure.yml` workflow    |
| `release-note-category.yml` | Validate a release-note category label is applied on a PR   |
| `release-note-category.js`  | The main script of the `release-note-category.yml` workflow |

```

Reply:
Thanks!